requirements.txt

```diff
@@ -29,11 +30,12 @@ typing-extensions==4.0.1
 Unidecode==1.2.0
 urllib3==1.26.19
 wheel
-zipp==3.19.1
+zipp==3.4.0
```

Comment:
Was this change required to add `pydoclint`?

Reply:
It is not required and has been revised back to zipp==3.19.1. I wonder how this happened.